### ISSUE TITLE
Support HTTP rather than HTTPS

### ISSUE THREAD

**Issue by vkarpov15** [keen/keen-tracking.js#82]
Looks like `keen-core` supports a 'protocol' option for clients, but keen-tracking enforces HTTPS even if the protocol is set to 'HTTP':

```javascript
const client = new Keen({
  projectId: 'PROJECT_ID',
  writeKey: 'WRITE_KEY',
  host: 'localhost',
  protocol: 'http'
});
```

Can you add support for HTTP? Would be very useful for using the keen client against an internal proxy that might not support HTTPS.

**Comment by dustinlarimer** [comment]
@vkarpov15 this should work as you describe now: https://github.com/keen/keen-core.js/blob/master/lib/index.js#L121. The value for `config.protocol` will overwrite the default if provided. Are you seeing a different behavior?

**Comment by vkarpov15** [comment]
Yes, the issue is that this lib uses the `https` module to make the request, regardless of the value of `protocol`. Here's the relevant code: https://github.com/keen/keen-tracking.js/blob/d72751421dd95fd561d4120879d9c3b40017e9ac/lib/record-events-server.js#L200

**Comment by dustinlarimer** [comment]
Ah ok, I see what you mean– that code only runs on the server, but I can see how that would cause issues for an internal proxy. Happy to keep this open for future feature dev, but to be totally transparent this might sit awhile before it gets picked up. Is this blocking development for you?

**Comment by vkarpov15** [comment]
Yeah that's exactly the context in which I ran into this issue, was experimenting with consolidating server-side API keys behind a gateway. Not a blocker at the moment, I'll put in a PR if that changes, just wanted to put this on your radar.

**Comment by dustinlarimer** [comment]
Sounds great, thanks – I appreciate your help in spotting this!

**Comment by dustinlarimer** [comment]
@vkarpov15 this has been released in `keen-tracking@1.4.0` thanks to @lukechilds [#hacktoberfest](https://github.com/keen/keen-tracking.js/labels/hacktoberfest) contribution. Thanks for reporting, and please let me know if you encounter any other issues or speed bumps.

**Comment by vkarpov15** [comment]
Nice! T-shirt for a PR sounds like a decent deal :)
